frontend/src/scenes/session-recordings/player/sidebar/PlayerSidebarSessionSummary.tsx

```diff
@@ -85,7 +151,7 @@ function SegmentMetaTable({ meta }: SegmentMetaProps): JSX.Element | null {
     return (
         <div className="grid grid-cols-2 gap-2 text-xs mt-2">
             <div className="flex items-center gap-1">
-                <IconKeyboard />
+                <IconKeyboard className={meta.key_action_count && meta.key_action_count > 0 ? 'text-success' : ''} />
```

Comment:
we use a convenience function that was called `cslx` but recently changed to make things like this easier to extend

not blocking since there's a single thing here 

ah, now called `cn` 

https://posthog.slack.com/archives/C0113360FFV/p1741762808953989